.github/workflows/release-content-to-candidate.yml

```diff
@@ -0,0 +1,70 @@
+name: Release ffmpeg-2204 to candidate
+
+on:
+  workflow_run:
+    workflows: ["Release ffmpeg-2204-sdk to candidate"]
+    types: [completed]
+    paths:
+      - "ffmpeg-2204/**"
+  workflow_dispatch:
+    
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+  issues: write
+
+jobs:
+  get-architectures:
+    if: ${{ github.event.workflow_run.conclusion == 'success' || github.event_name == 'workflow_dispatch' }}
```

Comment:
What is this doing at this point in the workflow? It looks redundant to me? :)

Reply:
This workflow will run when the `release-sdk-to-candidate` is successful. Now if the release fails, this condition is used to check that.